documentation/az.asciidoc

```diff
@@ -0,0 +1,46 @@
+:toc:
+toc::[]
+
+= Azure CLI
+
+The Azure CLI commandlet allows to install and use https://github.com/Azure/azure-cli[Azure CLI]. Currently, the command works only for Windows.
+
+*ATTENTION:*
+Currently this feature is new and therefore experimental.
+It may change in incompatible ways in the next releases until we reach a stable state.
+We hope that all is working fine for you.
+However, do not expect everything to work out of the box.
+In case you are facing issues (e.g. network problems with Cisco AnyConnect, etc.) please give us feedback so we can improve.
```

Comment:
It seems as if this sentence always gets copied from docker to other commandlets even though it does not make any sense here. Only Docker/Rancher have this VPN issue.

Reply:
I have now removed only the last sentence.